platzky_hotjar/plugin.py

```
@@ -0,0 +1,56 @@
+"""Platzky Hotjar plugin — injects Hotjar tracking code into page head."""
+
+import inspect
+from typing import Type, cast
```

Ruff (0.15.0)

[warning] `typing.Type` is deprecated, use `type` instead

(UP035)